.github/workflows/check.yml

```
@@ -89,7 +89,7 @@ jobs:
             cargo +${{ matrix.rust-toolchain }} test $BUILD_TYPE --no-fail-fast $FILTER
           fi
           cargo +${{ matrix.rust-toolchain }} bench --no-run
-      - uses: codecov/codecov-action@b9fd7d16f6d7d1b5d2bec1a2887e65ceed900238 # v4.6.0
+      - uses: codecov/codecov-action@5c47607acb93fed5485fdbf7232e8a31425f672a # v5.0.2
         with:
           file: lcov.info
```

```suggestion
          files: [lcov.info]
```

The v5 version of codecov switches from "file" to "files" according to the same notes that this PR references.